src/PrimeFuncPack.Core/core_objects/Objects.Tests/Objects.Tests.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <IsPackable>false</IsPackable>
+    <RootNamespace>PrimeFuncPack.Core.Objects.Tests</RootNamespace>
+    <AssemblyName>PrimeFuncPack.Core.Objects.Tests</AssemblyName>
```

Comment:
Shall we add Authors/Copyright tags? Tests are not supposed to be published as packages, but just to protect the sources

Reply:
Done